addons/hr_holidays/static/src/leave_stats/leave_stats.js

```diff
@@ -106,17 +120,25 @@ export class LeaveStatsComponent extends Component {
 
         const dateFrom = date.startOf("year");
         const dateTo = date.endOf("year");
-        this.state.leaves = await this.orm.readGroup(
+
+        const leaves = await this.orm.readGroup(
             "hr.leave",
             [
                 ["employee_id", "=", employee[0]],
                 ["state", "=", "validate"],
                 ["date_from", "<=", dateTo],
                 ["date_to", ">=", dateFrom],
             ],
-            ["holiday_status_id", "number_of_days:sum"],
+            [
+                "holiday_status_id",
+                "number_of_days",
+                "number_of_hours",
+                "leave_type_request_unit:array_agg",
+            ],
             ["holiday_status_id"]
         );
+
+        this.state.leaves = leaves;
```

Comment:
I'm guessing you can assign directly:
```js
        this.state.leaves = wait this.orm.readGroup(
            "hr.leave",
            [
                ["employee_id", "=", employee[0]],
                ["state", "=", "validate"],
                ["date_from", "<=", dateTo],
                ["date_to", ">=", dateFrom],
            ],
            [
                "holiday_status_id",
                "number_of_days",
                "number_of_hours",
                "leave_type_request_unit:array_agg",
            ],
            ["holiday_status_id"]
        );
```

Reply:
Indeed. I also noticed the dateFrom and dateTo in departmentLeaves was not locale formatted and there was some unneeded stuff in the test. I pushed an update so now it should all be good.